src/models/jobs.ts

```diff
@@ -9,15 +9,11 @@ type Job = {
     updatedAt?: Date
 }
 
-const tableName = 'logging';
+const tableName = 'jobs';
 
-const jobs = () => knex(tableName);
-const createJob = (parent : any, args : Job) => knex(tableName)
-    .insert(args)
-    .then(data => Promise.resolve({id: data[0]}));
-const deleteJob = (parent : any, args : Job) => knex(tableName)
-    .where(args)
-    .del()
-    .then(() => args);
+const jobs = () => select(tableName);
```

Comment:
@melnychukvitaliy it's suitable for TS use call of an abstract function in this way?

Reply:
I'm not sure about it. 

Actually, TS gives you types checking and types declaration and those are most important we have to pay attention and stick to this where it's possible. 
By using abstract methods we are starting using `any` type that is not good case for me.
I think, better just declare types for `create`, `delete`, `select` explicitly in each model and use knex library directly. 
In this case, we get rid of abstract model and be able to use knexjs methods as much as possible

Reply:
ok, got it